### DISCUSSION TITLE
[灌水 ]是我不懂现代艺术?

### DISCUSSION THREAD

**Discussion by Iincan** [share121/inter-knot#1140] [其他]
![Uploading 13526b8b99bb30269f0dc4ba79aa7781.png…]()
最近去看展。旧都陷落时，芭莱大厦被空洞侵蚀的程度还没有现在这么深。空洞调查员紧急抢救出一批展品。
看完只想说，调查员冒着生命危险抢救这种东西，真的有必要吗?
《抱邦布的男人(们)》《以骸背上的芭菜双子》是我最不喜欢的作品，感觉只是戏仿名作，顺便拍下芭莱老板的马屁。作者是金斯顿，伯德。据说这个人没有经过学院派的训练，只是一个临摹名画的画工，这都可以吗?

**Comment by 550w-1903** [comment]
233333
